defaults/main.yml

```diff
@@ -12,10 +12,18 @@ monasca_rsyslog_api_endpoint:
 monasca_rsyslog_api_auth:
 
 # List of package dependencies.
-monasca_rsyslog_packages:
```

Comment:
Do we have anything using this variable?

Reply:
Thanks for reviewing. Not afaik:
```
doug@xps15:~/git/ansible-role-monasca-rsyslog$ grep -ri monasca_rsyslog_packages
tasks/main.yml:  when: monasca_rsyslog_packages_install | bool and ansible_os_family == 'RedHat'
tasks/main.yml:  when: monasca_rsyslog_packages_install | bool and ansible_os_family == 'Debian'
defaults/main.yml:monasca_rsyslog_packages_install: True
README.md:`monasca_rsyslog_packages_install`: Flag to define whether package dependencies
```